front/app/containers/Admin/projects/project/analysis/Tasks/index.tsx

```diff
@@ -1,66 +1,167 @@
+          iconColorOnHover={colors.black}
+          onClick={() => setIsDropdownOpened(!isDropdownOpened)}
+          iconHeight="28px"
+          iconWidth="28px"
+        />
+      )}
+
+      <Dropdown
+        opened={isDropdownOpened}
+        width="300px"
+        onClickOutside={() => setIsDropdownOpened(false)}
+        right="20px"
+        top="60px"
+        content={
+          <Box p="12px">
+            {tasks?.data.map((task) => {
+              return (
+                <Box
+                  key={task.id}
+                  display="flex"
+                  flexDirection="column"
+                  gap="4px"
+                >
+                  <Box
+                    display="flex"
+                    justifyContent="space-between"
+                    alignItems="center"
+                    gap="8px"
+                  >
+                    <Text
+                      fontWeight="bold"
+                      fontSize="s"
+                      my="4px"
+                      style={{
+                        wordBreak: 'break-all',
+                      }}
+                    >
+                      {task.attributes.type}
+                    </Text>
+                    {task.attributes.type === 'auto_tagging_task' && (
+                      <Box
+                        as="span"
+                        bg={
+                          TagTypeColorMap[task.attributes.auto_tagging_method]
+                            .background
+                        }
+                        color={
+                          TagTypeColorMap[task.attributes.auto_tagging_method]
+                            .text
+                        }
+                        borderRadius="3px"
+                        py="4px"
+                        px="8px"
+                        w="fit-content"
+                      >
+                        {task.attributes.auto_tagging_method}
+                      </Box>
+                    )}
+                  </Box>
+                  {task.attributes.progress && (
+                    <StyledProgressBar
+                      progress={task.attributes.progress}
+                      color={colors.green400}
+                      bgColor={colors.grey100}
+                    />
+                  )}
+                  <Box
+                    as="span"
+                    bg={stateColorMap[task.attributes.state]}
+                    color="white"
+                    borderRadius="3px"
+                    py="4px"
+                    px="8px"
+                    w="fit-content"
+                    mb="4px"
+                  >
+                    {task.attributes.state}
+                  </Box>
+                  <Box display="flex" justifyContent="space-between">
+                    <span> Triggered at</span>
+                    <span>
+                      {formatDate(task.attributes.created_at)}{' '}
+                      {formatTime(task.attributes.created_at)}
```

Comment:
Relative time might be more useful to the user?

Reply:
Good point 👍